### PR TITLE
Add video quality information to vital

### PR DESCRIPTION
**Pull request by daniel-riehm**
Add a few new vital metadata tags containing information about the video encoding methods. These are mandatory if we ever want to encode a new KLV ST1108 packet - e.g. to encode GSD or VNIIRS.

### CODE REVIEW

**Comment by daniel-riehm** [comment]
(Linux failure unrelated)

**Comment by daniel-riehm** [comment]
jenkins test this please

**Comment by daniel-riehm** [comment]
@hdefazio

File: vital/types/metadata_traits.h [hunk 1]
```diff
@@ -49,6 +49,11 @@ TAG_TYPE( VITAL_META_VIDEO_URI, string_t );
 TAG_TYPE( VITAL_META_VIDEO_KEY_FRAME, bool );
 TAG_TYPE( VITAL_META_VIDEO_FRAME_NUMBER, uint64_t );
 TAG_TYPE( VITAL_META_VIDEO_MICROSECONDS, uint64_t );
+TAG_TYPE( VITAL_META_VIDEO_FRAME_RATE, double );
+TAG_TYPE( VITAL_META_VIDEO_BITRATE, uint64_t );
+TAG_TYPE( VITAL_META_VIDEO_COMPRESSION_TYPE, string_t );
+TAG_TYPE( VITAL_META_VIDEO_COMPRESSION_PROFILE, string_t );
+TAG_TYPE( VITAL_META_VIDEO_COMPRESSION_LEVEL, string_t );
```
Comment (hdefazio) [inline vital/types/metadata_traits.h:56]: Do these need to be alphabetical?

Comment (daniel-riehm) [reply vital/types/metadata_traits.h:56]: It doesn't seem like the rest of the list is trying for alphabetical, so I just went with putting related information next to each other (like how `IMAGE_WIDTH` and `IMAGE_HEIGHT` are next to each other).

**Pull request merged**